nemo/collections/asr/models/clustering_diarizer.py

```diff
@@ -508,6 +511,7 @@ def restore_from(
         override_config_path: Optional[str] = None,
         map_location: Optional[torch.device] = None,
         strict: bool = False,
+        safe: bool = False,
```

Comment:
@tango4j this won't work for diarization models, since the vad and speaker models are restored in the `_init_vad_model` and `_init_speaker_model` methods that don't take `safe` as additional param. Do you think we should fix it or maybe plan in the future?